pkgs/applications/editors/neovim/gnvim/default.nix

```diff
@@ -33,6 +33,10 @@ rustPlatform.buildRustPackage rec {
     EOF
   '';
 
+  buildPhase = ''
+    make build
+  '';
```

Comment:
```suggestion
  buildTargets = [ "build" ];
```

Reply:
If the `buildPhase` isn't changed, the `buildPhase` from `buildRustPackage` would've been used, right?

Reply:
Right.